evaluation/README.md

```diff
@@ -0,0 +1,81 @@
+# Evaluation and Prediction:
+
+The evaluation folder covers both model performance when evaluated and prediction done by thee model with test data.
+
+## LodBrok model
+
+### Lodbrok model evaluation
+
+- how to use? 
+    ``` python3 eval.py```
+
+- A pickle file is created to evaluate the model that we trained, this is stored in ```../data/spambrainz_dataset_eval.pickle``` file. 
+
+- It is then evaluated with the model we saved earlier in ```eval.py```. This will show how accurately the model fairs against the eval data. This is useful to avoid overfitting or underfitting the model.
+
+This is the result of evaluation done on eval_dataset:
+
+![](images/eval.png)
+
+### LodBrok model prediction
+
+- how to use? 
+    ``` python3 check_predict.py```
+
+- New dataset named ```../data/spambrainz_dataset_predict.pickle``` is used to test the model predictions and how accurate it is with new data.
+
+- The ```check_predict.py``` predicts the data given without any label specifying the type of editor account.
+
+- The results are shown in an easy-to-view manner by testing against the dataset via a confusion matrix.
+    - how to use? 
+        ``` python3 create_confusion_matrix.py```
+
+![](images/confusion_matrix.png)
```

Comment:
Please write the full interpretation of this image for laymen.

Reply:
I will add a reference image to understand it better.
Thanks!!